docs/modular-resources/upload/_index.md

```diff
@@ -43,9 +43,17 @@ Get the `org-id` for your {{< glossary_tooltip term_id="organization" text="orga
 viam module create --name <your-module-name> --org-id <your-org-id>
 ```
 
+This command creates a new `meta.json` metadata file in your current working directory, which serves as a template.
+
+If you later wish to make your module public, you can use the `viam module update` command to do so.
+
 {{% /tab %}}
 {{% tab name="Public" %}}
 
+{{% alert title="Important" color="note" %}}
+Once you upload a module as public, you cannot change it back to private.
```

Comment:
```suggestion
Once you update your module to mark it as public, you cannot change it back to private.
```

???? (upload vs update confusion) (but I like the clarification)

Reply:
haha this is me trying to use command names as copy verbs. I actually change visibility with `update`. But you're absolutely right. Conceptually, this should be upload.

Reply:
Ha! Maybe! (I agree with your comment before you edited it that the concept is indeed `upload`ing but the command is `update` )

Reply:
Haha right on. I think your copy change does convey this intricacy well!